lib/v2/bangumionline/index.js

```diff
@@ -0,0 +1,25 @@
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const url = 'https://bangumi.online/api/home';
+
+    const response = await got({
+        method: 'post',
+        url,
+        headers: {},
+    });
+
+    const list = JSON.parse(response.body).data.list;
+
+    const items = list.map((item) => ({
+        title: `${item.title_zh} - 第 ${item.volume} 集`,
+        description: `<img src='https:${item.cover}'>`,
```

Comment:
https://docs.rsshub.app/joinus/script-standard.html#template

Reply:
Thank you, art templates are already in use